testapp/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Mini App Test</string>
+    <string name="app_name">Mini App Sample App</string>
```

Comment:
Maybe just Sample App seems better. @corycaywood What do you think?

Reply:
I think "Mini App Sample App" is okay, or maybe just "Mini App Sample"? "Mini App Demo"? Something like that...

Reply:
Mini App Sample sounds okay too.